ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -695,9 +695,20 @@ literals:
   - ACTIVE
   - INACTIVE
 
-
-
-
+# New fields for lighting and fire systems.
+- motion_status
+- scene_command
```

Comment:
Scenes are likely to be free text, which is currently really only expected for a label.  Discuss...

Reply:
@pisuke please help

Reply:
I think we settled on scene_index_command?

Reply:
yes, scene_index_command was the agreed direction